python/vowpalwabbit/pyvw.py

```diff
@@ -564,7 +564,7 @@ def __init__(self, label, cost=0., partial_prediction=0., probability=0.):
             self.costs.append(wc)
 
     def __str__(self):
-        return ' '.join([str(c.label) + ':' + str(c.cost) for c in self.costs])
+        return ' '.join([str(c.action) + ':' + str(c.cost) + ':' + str(c.probability) for c in self.costs])
```

Comment:
Could you change this to using `format`? Will make the output a bit clearer

Reply:
Sure!